packages/mdc-textfield/foundation.js

```diff
@@ -202,9 +198,8 @@ class MDCTextfieldFoundation extends MDCFoundation {
    * Fires when animation transition ends, performing actions that must wait
    * for animations to finish.
    * @param {!Event} evt
-   * @private
    */
-  transitionEnd_(evt) {
+  transitionEnd(evt) {
```

Comment:
I'm kind of surprised that we'd be publicizing this method in particular... would there be a better name for this that would give better context to when it'd make sense to directly invoke it?

Reply:
Gave this and setBottomLineTransformOrigin more semantic names.